vmdb/app/models/configured_system_foreman.rb

```diff
@@ -1,2 +1,11 @@
 class ConfiguredSystemForeman < ConfiguredSystem
+
+  before_save :update_installing
```

Comment:
Instead of a Rails hook (which always freak me out), maybe override raw_installing=.   This is how we did raw_power_state= for vms .  Additonally,  we also blocked power_state=,  so you may want to block installing=.

However, the reason we did that for power_state was because we needed the raw state to make decisions,  but needed to keep power_state normalized. Why do you need to store raw_installing at all (especially since it just looks like the string 'true')?